### DISCUSSION TITLE
Conflict with other tracking apps?

### DISCUSSION THREAD

**Discussion by SiggiKeller** [owntracks/android#1833] [Q&A]
Question of a newbe: Will OwnTracks conflict with other tracking apps?
I run OwnTracks on my Samsung S22+ and it works. Not perfect, but I'm getting some results.
Yesterday I went hiking and I used Komoot for navigation. OwnTrack stopped working and I had manually to restart the app after stopping Komoot. Is this a normal behaviour?

**Comment by growse** [comment]
It shouldn't conflict. If anything, using other apps which request locations should help OT be better, as the device OS will deliver more accurate locations more frequently to OT.

We've had reports of reliability issues with other Samsung users though - it seems they're pretty aggressive in disabling other apps receiving locations / scheduling their services when there's something else more actively running in the foreground.
